jenkins/jobs/integration_tests.pipeline

```diff
@@ -73,7 +73,9 @@ pipeline {
           try {
             withCredentials([usernamePassword(credentialsId: 'airshipci_city_cloud_openstack_credentials', usernameVariable: 'OS_USERNAME', passwordVariable: 'OS_PASSWORD')]) {
               withCredentials([sshUserPrivateKey(credentialsId: 'airshipci_city_cloud_ssh_keypair', keyFileVariable: 'AIRSHIP_CI_USER_KEY')]) {
-                sh "./jenkins/scripts/integration_test.sh"
+                withCredentials([githubToken(credentialsId: 'airshipci_city_cloud_github_token', keyFileVariable: 'GITHUB_TOKEN')]) {
```

Comment:
Is the `GITHUB_TOKEN` value a string ? If yes, I think you need to use [Secret text type](https://www.jenkins.io/doc/book/using/using-credentials/) of credentials here. For that you will need: 
1. Use `string` instead of `githubToken`.  You can see example [here](https://www.jenkins.io/doc/pipeline/steps/credentials-binding/). Is there githubToken credential type in Jenkins?
2. Use `variable:` instead of `keyFileVariable:`    since the token it is not a file but a string.

Reply:
Fixed

Reply:
Fixed